README.md

```diff
@@ -29,20 +29,19 @@ Takes a version and array of ranges, returns the greatest satisfied range. Range
 
 MIT
 
-[range-support]: https://github.com/phated/sver-compat#range-support
 
-[downloads-image]: http://img.shields.io/npm/dm/semver-greatest-satisfied-range.svg
+<!-- prettier-ignore-start -->
+[downloads-image]: https://img.shields.io/npm/dm/semver-greatest-satisfied-range.svg?style=flat-square
 [npm-url]: https://www.npmjs.com/package/semver-greatest-satisfied-range
-[npm-image]: http://img.shields.io/npm/v/semver-greatest-satisfied-range.svg
-
-[travis-url]: https://travis-ci.org/gulpjs/semver-greatest-satisfied-range
-[travis-image]: http://img.shields.io/travis/gulpjs/semver-greatest-satisfied-range.svg?label=travis-ci
+[npm-image]: https://img.shields.io/npm/v/semver-greatest-satisfied-range.svg?style=flat-square
 
-[appveyor-url]: https://ci.appveyor.com/project/gulpjs/semver-greatest-satisfied-range
-[appveyor-image]: https://img.shields.io/appveyor/ci/gulpjs/semver-greatest-satisfied-range.svg?label=appveyor
+[ci-url]: https://github.com/gulpjs/semver-greatest-satisfied-range/actions?query=workflow:dev
+[ci-image]: https://img.shields.io/github/workflow/status/gulpjs/semver-greatest-satisfied-range/dev?style=flat-square
 
 [coveralls-url]: https://coveralls.io/r/gulpjs/semver-greatest-satisfied-range
-[coveralls-image]: http://img.shields.io/coveralls/gulpjs/semver-greatest-satisfied-range/master.svg
+[coveralls-image]: https://img.shields.io/coveralls/gulpjs/semver-greatest-satisfied-range/master.svg?style=flat-square
+<!-- prettier-ignore-end -->
 
-[gitter-url]: https://gitter.im/gulpjs/gulp
-[gitter-image]: https://badges.gitter.im/gulpjs/gulp.svg
+<!-- prettier-ignore-start -->
+[range-support]: https://github.com/phated/sver-compat#range-support
```

Comment:
This needs to be updated to the official `sver` module. The documentation about "sver-compat" need to be updated to reference the "sver" project.